target_postgres/json_schema.py

```diff
@@ -1,6 +1,7 @@
 from copy import deepcopy
 import json
 import re
+import decimal
```

Comment:
This isn't documented anywhere, so I wouldn't expect you to know this one, but we try to alphabetize our `import`s. The breakdown is:

- Python
- 3rd Party
- Our code